src/less/components/toast.less

```diff
@@ -98,7 +96,7 @@
 			}
 		}
 
-		.material-icons {
+		i {
```

Comment:
Probably a bit too generic since it is okay to pass html as content to the toast